project/src/components/app-pages/main-page/main-page.tsx

```diff
@@ -10,6 +17,8 @@ type MainPageProps = {
 
 function MainPage(props: MainPageProps): JSX.Element {
   const {title, genre, year} = props;
+  const selectedGenre = useSelector(getSelectedGenre);
```

Comment:
например такое название селектора:
```suggestion
  const selectedGenre = useSelector(selectorForSelectedGenre);
```